packages/shared.net/src/createClientContext.ts

```diff
@@ -31,22 +31,23 @@ export function createClientContext<
   ontology: T,
   stack: string,
   tokenProvider: () => Promise<string> | string,
-  additionalUserAgent?: `${string} (${string})`,
+  userAgent: string,
   fetchFn: typeof globalThis.fetch = fetch,
 ): ClientContext<T> {
   if (stack.length === 0) {
     throw new Error("stack cannot be empty");
   }
+
+  const wireUserAgent = `${userAgent} ${ontology.metadata.userAgent}`;
```

Comment:
Examples here would be

```typescript
"@osdk/client/0.0.0 @osdk/cli/0.0.0"
"@osdk/legacy-client/0.0.0 @osdk/foundry-sdk-generator/0.0.0"
```

Do we want to de-package-name-ify them? This would make splitting on `/` safer if we ever did that.

```
@osdk/cli -> osdk-cli
@osdk/legacy-client -> osdk-legacy-client
@osdk/client -> osdk-client
@osdk/foundry-sdk-generator -> osdk-foundry-sdk-generator
```